code/modules/cargo/packs/ammo.dm

```diff
@@ -70,6 +70,12 @@
 	cost = 500
 	contains = list(/obj/item/ammo_box/a12g/slug)
 
+/datum/supply_pack/ammo/techshells
+	name = "Unloaded Shotgun Technological Shells Crate"
+	desc = "Contains a box of 7 versatile tech shells, capable of producing a variety of deadly effects for any situation. Some assembly required."
+	cost = 175
```

Comment:
20 per buckshot or slug
25 per tech shell..
I would prefer if they cost more, the crafting ingredients aren't enough to affect the balance

Reply:
upped it by another 5, it's now 30 per tech shell.